src/components/Tabs/README.md

```diff
@@ -1,19 +1,178 @@
+    ]}
+/>
+`}
+>
+    <UIKit.Tabs
+        activeTab="first"
+        items={[
+            {id: 'first', title: 'Third Tab'},
+            {id: 'second', title: 'Second Tab'},
+            {id: 'third', title: 'Third Tab', disabled: true},
+        ]}
+    />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+const [activeTab, setActiveTab] = React.useState('second');
+
+return (
+  <Tabs
+    activeTab={activeTab}
+    onSelectTab={(tabId) => setActiveTab(tabId)}
+    items={[
+      {id: 'first', title: 'Third Tab'},
+      {id: 'second', title: 'Second Tab'},
+      {id: 'third', title: 'Disabled Tab', disabled: true},
+    ]}
+  />
+);
+```
+
+<!--/GITHUB_BLOCK-->
+
+Also `Tabs` has dedicated component `Tabs.Item` for an item.
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+const [activeTab, setActiveTab] = React.useState('second');
+
+return (
+  <Tabs activeTab={activeTab}>
+    <Tabs.Item id={'first'} title="Third Tab" onClick={(tabId) => setActiveTab(tabId)} />
+    <Tabs.Item id={'second'} title="Active Tab" onClick={(tabId) => setActiveTab(tabId)} />
+  </Tabs>
+);
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Size
+
+To control the size of the `Tabs` use the `size` property. Default size is `m`.
+
+<!--LANDING_BLOCK
+
+<ExampleBlock
+    code={`
+<Tabs
+    activeTab="second"
+    size="m"
+    items={[
+        {id: 'first', title: 'medium 1'},
+        {id: 'second', title: 'medium 2'},
+    ]}
+/>
+
+<Tabs
+    activeTab="second"
+    size="l"
+    items={[
+        {id: 'first', title: 'large 1'},
+        {id: 'second', title: 'large 2'},
+    ]}
+/>
+
+<Tabs
+    activeTab="second"
+    size="xl"
+    items={[
+        {id: 'first', title: 'extra large 1'},
+        {id: 'second', title: 'extra large 2'},
+    ]}
+/>
+`}
+>
+    <UIKit.Tabs
+        activeTab="second"
+        size="m"
+        items={[
+            {id: 'first', title: 'medium 1'},
+            {id: 'second', title: 'medium 2'},
+        ]}
+    />
+    <UIKit.Tabs
+        activeTab="second"
+        size="l"
+        items={[
+            {id: 'first', title: 'large 1'},
+            {id: 'second', title: 'large 2'},
+        ]}
+    />
+    <UIKit.Tabs
+        activeTab="second"
+        size="xl"
+        items={[
+            {id: 'first', title: 'extra large 1'},
+            {id: 'second', title: 'extra large 2'},
+        ]}
+    />
+</ExampleBlock>
+
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Tabs
+    activeTab="second"
+    size="m"
+    items={[
+        {id: 'first', title: 'medium 1'},
+        {id: 'second', title: 'medium 2'},
+    ]}
+/>
 
-### Tabs PropTypes
+<Tabs
+    activeTab="second"
+    size="l"
+    items={[
+        {id: 'first', title: 'large 1'},
+        {id: 'second', title: 'large 2'},
+    ]}
+/>
+
+<Tabs
+    activeTab="second"
+    size="xl"
+    items={[
+        {id: 'first', title: 'extra large 1'},
+        {id: 'second', title: 'extra large 2'},
+    ]}
+/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Properties
```

Comment:
Let's rename to "Tabs properties" to be able to distinguish from `Tabs.Item` properties